src/LocalMixedGraphSpectralTargets.cpp

```diff
@@ -581,9 +581,9 @@ void LocalMixedGraphSpectralTargets::ComputeEdgeTargets(
                                              face_nbh_dofs, colMapper);
             sec_D.ReduceSend(iface, Dloc);
         }
-        else // global boundary face
+        else // global boundary face or only 1 dof on face
         {
-            mfem::SparseMatrix empty_matrix;
```

Comment:
Did the difference between these `empty_matrix` and `SparseIdentity(0)` cause issues?  Just curious.

Reply:
Yes. I think the problem is when we send/receive the matrix in SharedEntityCommunication, it will check the nnz (to know how many data to send), which is the last entry of I. But if we initialize a SparseMatrix without doing anything, I points to NULL.